.vscode/tasks.json

```diff
@@ -63,6 +63,20 @@
 				"showReuseMessage": false,
 				"clear": true
 			}
+		},
+		{
+			"label": "generate-certs",
+            "type": "shell",
+            "command": "mkdir -p ~/tls/cert && openssl req -x509 -newkey rsa:4096 -keyout ~/tls/cert/tls.key -out ~/tls/cert/tls.crt -days 365 -nodes -subj \"/CN=localhost\" && export TLSCertDir=~/tls/cert",
+            "problemMatcher": [],
+			"windows": {
+				"command": "if not exist %USERPROFILE%\\tls\\cert mkdir %USERPROFILE%\\tls\\cert && openssl req -x509 -newkey rsa:4096 -keyout %USERPROFILE%\\tls\\cert\\tls.key -out %USERPROFILE%\\tls\\cert\\tls.crt -days 365 -nodes -subj \"/CN=localhost\" && set TLSCertDir %USERPROFILE%\\tls\\cert"
+			}
```

Comment:
Question about this.... When you fire this up in the debugger, does the webhook actually get called by Kubernetes? How does that work?

Reply:
Usually with Kubernetes webhooks, the only way to **really** test them is to deploy the webhook to the cluster. So what I see most people do is **turn off** the webhook in debugging scenarios.

Reply:
yes, that makes sense, reverted changes. now passing in "" for cert-dir when debugging.